buildstockbatch/schemas/v0.1.yaml

```diff
@@ -33,9 +33,9 @@ aws-job-environment:
   memory: int(min=1024, required=False)
 
 aws-emr-spec:
-  master_instance_type: str(required=False)
-  slave_instance_type: str(required=False)
-  slave_instance_count: int(min=1, required=False)
+  manager_instance_type: str(required=False)
+  worker_instance_type: str(required=False)
+  worker_instance_count: int(min=1, required=False)
```

Comment:
We shouldn't be changing old schemas as the only function they serve now is to keep a record of previous input formats.

Reply:
Makes sense, addressed in https://github.com/NREL/buildstockbatch/pull/205/commits/517049db72bbd2f5ab210d85db08a739e421dd86